imu_extraction.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://humanisingautonomy.com/schemas/imu_extraction.schema.json",
+  "title": "IMU Extraction",
+  "description": "The IMU data extracted from a video source",
```

Comment:
Suggestion:
"description": "The IMU data extracted and represented into a timeline from a Nextbase dashcam video",